### PR TITLE
update the Volume2 transaction pages

### PR DESCRIPTION
**Pull request by JohnMoehrke**
supports #475 

Put formal titles on the transaction pages

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed
- [x] I have selected a committee co-chair to review the PR

### CODE REVIEW

File: ITI/TF/Volume2/ITI-118.html
Comment (MaryLJ) [inline ITI/TF/Volume2/ITI-118.html:1]: Should id and xaction number be 118 vs. 117 and the name should be Find Existing Appointments

**Review by MaryLJ — COMMENTED** [review]
iti-118 name is Find Existing Appointments I think, and the id and transaction number in the ITI-118 html files have ITI-117 vs. ITI-118

File: ITI/TF/Volume2/ITI-118.html
Comment (JohnMoehrke) [reply ITI/TF/Volume2/ITI-118.html:1]: I am using the details from the ITI.Scheduling publication.

Ah, I see that publication says somethign different on the other.html page. We will fix the other.html page in the future.

**Comment by JohnMoehrke** [comment]
> iti-118 name is Find Existing Appointments I think, and the id and transaction number in the ITI-118 html files have ITI-117 vs. ITI-118

the other.html page is wrong. Lets leave it wrong in the publication, but I will fix it for future publications.

**Comment by JohnMoehrke** [comment]
> > iti-118 name is Find Existing Appointments I think, and the id and transaction number in the ITI-118 html files have ITI-117 vs. ITI-118
> 
> the other.html page is wrong. Lets leave it wrong in the publication, but I will fix it for future publications.

well, it is different in more than that place... Either way, lets leave it as is for formal pub, and fix it in the future.

**Comment by JohnMoehrke** [comment]
> > > iti-118 name is Find Existing Appointments I think, and the id and transaction number in the ITI-118 html files have ITI-117 vs. ITI-118
> > 
> > 
> > the other.html page is wrong. Lets leave it wrong in the publication, but I will fix it for future publications.
> 
> well, it is different in more than that place... Either way, lets leave it as is for formal pub, and fix it in the future.

Seems Find is used more than Search. Search only shows up once. So I will change to Find.

**Comment by JohnMoehrke** [comment]
> > > > iti-118 name is Find Existing Appointments I think, and the id and transaction number in the ITI-118 html files have ITI-117 vs. ITI-118
> > > 
> > > 
> > > the other.html page is wrong. Lets leave it wrong in the publication, but I will fix it for future publications.
> > 
> > 
> > well, it is different in more than that place... Either way, lets leave it as is for formal pub, and fix it in the future.
> 
> Seems Find is used more than Search. Search only shows up once. So I will change to Find.

Should now be correct to "Find Existing Appointments"

Note that the IG will stay "Search Patient Appointments" on the index.html page, it is correct everywhere else. - This will be fixed in a future publication.

**Comment by JohnMoehrke** [comment]
@MaryLJ this should now be as close as we can get without rebuilding the ITI.Scheduling supplement.  The build of ITI.Scheduling is already integrated into publications repo, so it is very hard to change at this point.

**Review by MaryLJ — APPROVED** [review]
OK

**Pull request merged**